pkg/neorpc/filters.go

```diff
@@ -48,6 +51,43 @@ type (
 		Type   *mempoolevent.Type `json:"type,omitempty"`
 	}
 )
+type ValidFilter interface {
```

Comment:
Change the commit title to something like `neorpc: add WS notification filter IsValid functionality`, otherwise it's not that clear what's the `filter` in this scope.